pkg/acsengine/template_generator.go

```diff
@@ -1145,6 +1145,21 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 		"EnablePodSecurityPolicy": func() bool {
 			return helpers.IsTrueBoolPointer(cs.Properties.OrchestratorProfile.KubernetesConfig.EnablePodSecurityPolicy)
 		},
+		"ACIConnectorAddonEnabled": func() bool {
```

Comment:
I don't think we need those functions, I think we can do `{{if .OrchestratorProfile.KubernetesConfig.ACIConnectorAddonEnabled}}` directly in the template just like we use .MasterProfile for https://github.com/Azure/acs-engine/blob/master/parts/k8s/kubernetesmastervars.t#L257